README.md

```diff
@@ -249,6 +249,22 @@ let t = of_json (Json.parseOrRaise {|{"a": 42}|})
 (* t = { a = 42; b = None; } *)
 ```
 
+#### `[@json.drop_default]`: drop default values from JSON
+
+When a field has `[@option]` attribute one can use `[@json.drop_default]`
+attribute to make the generated `to_json` function to drop the field if it's
+value is `None`:
+
+```ocaml
+type t = {
+  a: int;
+  b: string option [@json.option] [@json.drop_default];
```

Comment:
Is it `option` (line 254) or `json.option`?

Reply:
It’s both, in ppxlib you register attributes fully qualified but they can be used both ways.